mmv1/products/serviceusage/api.yaml

```diff
@@ -103,6 +103,10 @@ objects:
         input: true
         description: |
           The limit on the metric, e.g. `/project/region`.
+
+          ~> Make sure that `limit` is in a format that doesn't start with `1/` or contain curly braces.
+          E.g. use `/project/user` instead of `1/{project}/{user}`.
+          The format used in the provider differs from format used by the gcloud CLI (due to the feature being in beta).
```

Comment:
Is the feature being in beta the reason for this difference? It seems like we've just made different choices, and when the feature comes out of beta we will still have the difference

Reply:
I must admit I assumed that the gcloud CLI is using a 'newer' version of the API and that things would converge in future as it comes out of beta, but that's not necessarily the case. My motivation for highlighting that the feature is in beta was to acknowledge that there's conflicting information out there about the unit format (different format used for gcloud, and [API docs saying to use a format that is currently incorrect for the API used by the provider](https://cloud.google.com/service-usage/docs/reference/rest/v1beta1/services.consumerQuotaMetrics.limits.consumerOverrides#:~:text=An%20example%20unit%20would%20be%3A%201/%7Bproject%7D/%7Bregion%7D%20Note%20that%20%7Bproject%7D%20and%20%7Bregion%7D%20are%20not%20placeholders%20in%20this%20example%3B%20the%20literal%20characters%20%7B%20and%20%7D%20occur%20in%20the%20string)).

Would removing the final sentence from the description be ok?

Reply:
Very strange... Yeah I think removing the final sentence would be the way to go, but it's super strange to me that the API doesn't accept the form it documents